src/components/Main/DraftsPage/DraftsPage.sass

```diff
@@ -0,0 +1,18 @@
+.drafts-page
+  padding: 40px 1rem
+  width: 80%
+  h3
+    margin-bottom: 20px
+  &-container
+    width: 100%
+    display: flex
+    flex-wrap: wrap
+    justify-content: space-between
+    align-items: start
+    .draft-item
```

Comment:
Better have a separate class .drafts-page-item
To separate Element in BEM, I use __ e.g. .drafts-page__item
To separate Modifier -- e.g. .drafts-page--narrow

Reply:
done